helm/templates/frontend-env-config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: frontend-env-config
+data:
+  env-config.js: |
+    window.process_env = {
+      SUBSTRATE_TELEMETRY_URL: "ws://feed.telemetry.parity-stg.parity.io/feed",
```

Comment:
I guess untimately this value should be injected from the values file, so that we can configure it for live etc

Reply:
this could also be a variable in the values.yaml file, but since we are not going to change every now and then I think keeping it like this would be fine. Otherwise we can just make a new variable and reference it here.